docs/integration.rst

```diff
@@ -1011,3 +1011,13 @@ QuboleFileSensor
 ''''''''''''''''
 
 .. autoclass:: airflow.contrib.sensors.qubole_sensor.QuboleFileSensor
+
+QuboleCheckOperator
+'''''''''''''''''''
+
+.. autoclass:: airflow.contrib.operators.qubole_check_operator.QuboleCheckOperator
+
+QuboleValueCheckOperator
+''''''''''''''''''''''''
+
+.. autoclass:: airflow.contrib.operators.qubole_check_operator.QuboleValueCheckOperator
```

Comment:
While I agree with the first case that we should have operators/hooks/sensors should be in the `docs/code.rst`, I don't fully agree to the Second.

It is cleaner and easier for someone working on AWS, GCP to quickly navigate to that `integrations` section and look for the features for that cloud on Airflow.

Reply:
Fair enough, I can see that.

Reply:
Having looked at how this is rendered how about a middle ground - we list the classes in this doc, but just as links, rather than including the code docstrings?

For example, in this screen shot the list and short description would stay, but the EmrAddStepsOperator wouldn't be in this doc?

<img width="726" alt="screen shot 2018-11-06 at 15 11 25" src="https://user-images.githubusercontent.com/34150/48073272-50d22380-e1d6-11e8-801a-5a2e5401dbc3.png">